src/apps/dashboard/routes/users/parentalcontrol.tsx

```diff
@@ -1,6 +1,6 @@
-import type { AccessSchedule, ParentalRating, UserDto } from '@jellyfin/sdk/lib/generated-client';
+import { UnratedItem, type AccessSchedule, type ParentalRating, type UserDto } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
`UnratedItem` should be imported using the full path for tree-shaking since it isn't a type.